guides-cli/src/main/java/io/micronaut/guides/cli/GradleMavenCommandMacro.java

```diff
@@ -0,0 +1,51 @@
+package io.micronaut.guides.cli;
+
+import io.micronaut.core.io.ResourceLoader;
+import io.micronaut.guides.core.Guide;
+import io.micronaut.guides.core.GuidesOption;
+import io.micronaut.guides.core.GuidesTemplatesConfiguration;
+import io.micronaut.guides.core.MacroSubstitution;
+import io.micronaut.guides.core.asciidoc.AsciidocMacro;
+import jakarta.inject.Singleton;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Singleton
+public class GradleMavenCommandMacro extends GradleMavenTabs implements MacroSubstitution {
+    public static final String MACRO = "gradle-maven-command";
+    public static final String START = "<code class=\"language-bash hljs\" data-highlighted=\"yes\"  style=\"white-space: pre-line;\">";
+    public static final String END = "</code>";
+    public static final String GRADLE_REGEX = "gradle=([^,]+(?:,[^,=]+)*),maven=";
```

Comment:
we should not need to do this regular expression logic. We already put in place attribute parsing logic for asciidoc macros.